allennlp/tests/tools/drop_eval_test.py

```diff
@@ -72,3 +72,31 @@ def test_multi_span_overlap_in_incorrect_cases(self):
         # macro-averaging F1 over spans
         assert get_metrics(["ottoman", "Kantakouzenous"],
```

Comment:
Can you add another test that has "answer" in the answer string and fails before this PR, just to be sure we don't have a regression in the future?

Reply:
I'm not sure if there were other issues that are fixed here, too; in general, you want a failing test case for every bug, that the PR then fixes.  This makes sure that we understand what the actual cause of the underlying bug was, and lets us be confident that we fixed it and it will stay fixed.

Reply:
There was only "answer" bug that was causing the empty prediction span bug. I have added a test for that.